model.js

```diff
@@ -9,16 +9,33 @@ function returnTopics(){
     })
 }
 
-
+function returnArticleComments(articleId){
+    return db.query("SELECT * FROM articles WHERE article_id = $1;", [articleId])
+    .then(({rows})=>{
+        if(rows.length === 0){
+            return Promise.reject({
+                status: 404,
+                msg: `No article found for article_id: ${articleId}`,
+              })
+        }
+        else{
+            return db.query("SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC;" ,[articleId])
```

Comment:
we can do this in a single query using a JOIN on the comments table... have another look at the notes on SQL 👀

Reply:
Thanks, I'll look into that.